scripts/release/strip-theme-links.js

```diff
@@ -17,8 +17,17 @@ const readmeFile = path.resolve(rootDir, 'README.md');
 const readme = await readFile(readmeFile, 'utf8');
 await writeFile(
   readmeFile,
-  readme.replace(
-    /https:\/\/cdn.simpleicons.org\/(.+)\/000\/fff/g,
-    `https://raw.githubusercontent.com/simple-icons/simple-icons/${LINKS_BRANCH}/icons/$1.svg`,
-  ),
+  readme
+    .replace(
+      /https:\/\/cdn.simpleicons.org\/(.+)\/000\/fff/g,
+      `https://raw.githubusercontent.com/simple-icons/simple-icons/${LINKS_BRANCH}/icons/$1.svg`,
+    )
+    .replace(
+      /\[!(NOTE|TIP|IMPORTANT|WARNING|CAUTION)\](?!\()/g,
+      function (str, $0) {
+        const capital = $0.substr(0, 1);
+        const body = $0.substr(1).toLowerCase();
+        return `**${capital + body}**`;
+      },
+    ),
```

Comment:
This `strip-theme-links.js` script is only used for strip theme links.

Reply:
Have updated the PR to reflect file renaming. Thanks @LitoMore!